docs/source/code_samples/data/version.py

```diff
@@ -0,0 +1,24 @@
+# ... client initialization left out
+data_client = client.data
+
+# Creates a new dataset (permissions default to private)
+dataset = data_client.create_dataset("My New Dataset")
+dataset_id = dataset["id"]
+
+# Uploads a file to it
+data_client.upload(dataset_id, "my_file.json")
+
+# Make the dataset public
+data_client.update_dataset(dataset_id, public=True)
```

Comment:
Why are you including this in the example?  It seems tangential.

Reply:
Agreed.

Reply:
👍